apps/web-portal/index.mjs

```diff
@@ -1,9 +1,50 @@
-import {WebPortalExpressApp} from './lib'
-import {config} from './config.mjs'
+import {createLogger} from '@zorko-io/util-logger'
+import {WebPortalExpressApp, setLogger} from './lib'
+import {config} from './config'
+
+const logger = createLogger({})
+
+setLogger(logger)
 
 const app = new WebPortalExpressApp({
-  config
```

Comment:
pass `logger` instance to the constructor, the same could be done for `process`

Reply:
the first version was with context passing but it looks a little bit complicated and I don't see advantages.
Why we are not reusing the logger defined in one place? In the case of using context passing, we need to use it in useCases and others

Reply:
done